platform/feature-api/communication/src/remote-pod/tests/node/bubblewrap.test.ts

```diff
@@ -0,0 +1,57 @@
+import { podBubblewrapClasses, podBubblewrap } from "../../remote";
+import {
+    NamedNode,
+    BlankNode,
+    Literal,
+    Variable,
+    DefaultGraph,
+    Quad,
+} from "@polypoly-eu/api";
+
+describe("Bubblewrap sanity check", () => {
+    test("should have the right number of classes", () => {
+        expect(Object.keys(podBubblewrapClasses).length).toBe(6);
+    });
+});
+
+const testInstances = [
+    [NamedNode, "https://example.org/n"],
+    [BlankNode, "https://example.org/n"],
+    [Literal, "Privacy rocks"],
+    [Variable, "Privacy rocks invariably"],
+    [DefaultGraph, "https://polypoly.coop/CDS"],
+];
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function testRoundtrip(anObject: any, aClass: any, msg: string): void {
```

Comment:
Naming nit: At first I thought this was a specific test, when it's more of a.... `assertRoundtrip` used in other tests, right?

Reply:
Right.